mubench.pipeline/requirements.py

```diff
@@ -74,3 +77,56 @@ def __init__(self):
 
     def check(self):
         pass
+
+class CPUCountRequirement(Requirement):
+    MIN_CPU_COUNT = 2
+
+    def __init__(self):
+        super().__init__("CPUs >= {}".format(self.MIN_CPU_COUNT))
+
+    def check(self):
+        cpu_count = self._get_cpu_count()
+        if cpu_count < self.MIN_CPU_COUNT:
+            raise ValueError("Only {} CPUs available.".format(cpu_count))
+
+    def _get_cpu_count(self):
+        if _in_container():
+            return self._get_container_cpu_count()
+        else:
+            return psutil.cpu_count(logical=False)
```

Comment:
I just changed it. Still, I added another requirement to pretty print memory size, which is also used inside the container. However, `psutils` should not be needed in the container anymore.
Now that brings up another problem: we have different requirements. Now we would need two separate requirements files. I'm not sure if it wouldn't be more sensible to just build the container with `psutils` anyway.